src/bp/ui-studio/src/web/views/OneFlow/diagram/ContentForm/index.tsx

```diff
@@ -105,6 +106,9 @@ const ContentForm: FC<Props> = ({
         {!!contentFields && (
           <Contents.Form
             bp={{ axios, mediaPath: `${window.BOT_API_PATH}/media` }}
+            customFields={{
+              text: props => <TextField {...props} />
```

Comment:
This prevents using the "text" property elsewhere. Since TextField is a component in the studio, I suggest adding it as an official component on ui-shared, for example text_variations. Otherwise, move the component on the builtin module, and use the module/component system  to load that component